lib/core/src/lib/layout/components/layout-container/layout-container.component.scss

```diff
@@ -16,18 +16,19 @@ adf-layout-container {
     }
 }
 
-.adf-sidenav--hidden {
-    visibility: hidden !important;
-    width: 0 !important;
-    transform: unset !important;
-    opacity: 0 !important;
-}
-
 .adf-layout-container {
     display: block;
     width: 100%;
     height: 100%;
     overflow: hidden;
+
+    /* stylelint-disable declaration-no-important */
+    .adf-sidenav--hidden {
+        width: 0 !important;
```

Comment:
Do we really neede thse 3 importants? Is not there a way to remove them?

Reply:
Unfortunatelly these properties are set in inline styles by angular material. I can only override them with important. I can't any other possibility.

Reply:
Hmm did you try to use mat-sidenav API? If you look at usage of this class in HTML then you can see this fragment: 
<img width="1004" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/820e85bb-e5dd-4612-a306-f99443856196">

So I guess you can even remove that class completely probably and use [opened] which exists already in that HTML to hide this sidenav (so opened can be set to false).

Thanks that there won't be custom styles so no need for !important as well.

Reply:
thank you